extension/src/types.ts

```diff
@@ -81,15 +81,15 @@ export class Extension {
     }
 }
 
-export enum MessageType {
+export enum MessageType { // those values must be in sync with AWS lambda routes of servrless.yml
     DEBUG_MESSAGE,
-    CHANGE_ROOM,
+    CHANGE_ROOM = "join-room",
```

Comment:
We should teenage the even value too.

Reply:
teenage ?

Reply:
Sorry. My phone autocorrect changed my sentence 

I wanted to say that we should also use JOIN_ROOM as the event value

Reply:
ok, but is it fine to have API Gateway routes/urls with uppercase (cf serverless.yml) ?